src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -90,7 +90,7 @@
 #include "ngraph/op/fused/gru_cell.hpp"
 #include "ngraph/op/fused/lstm_cell.hpp"
 #include "ngraph/op/fused/mvn.hpp"
-#include "ngraph/op/fused/normalize.hpp"
+#include "ngraph/op/fused/normalize_l2.hpp"
```

Comment:
All entries of "case OP_TYPEID::Normalize" in this file should be changed to "case OP_TYPEID::NormalizeL2".  Otherwise you will see INTELGPU backend build errors. Also decomposition for NormalizeL2 will not work correctly on this backend.

Reply:
Done, thank you